packages/api/src/objectSet/ObjectSet.test.ts

```diff
@@ -34,13 +43,32 @@ describe("ObjectSet", () => {
     aggregate: vi.fn(() => {
       return {};
     }),
-  } as any as EmployeeApiTest.ObjectSet;
+    pivotTo: vi.fn(() => {
+      return fauxObjectSet;
+    }),
+  } as any as $ObjectSet<Q>;
+
+  return fauxObjectSet;
+}
+
+describe("ObjectSet", () => {
+  const fauxObjectSet = createMockObjectSet<EmployeeApiTest>();
 
   describe("normal", () => {
     test("select none", async () => {
       const result = await fauxObjectSet.fetchPage();
       expectTypeOf<typeof result.data[0]>().toEqualTypeOf<
-        Osdk.Instance<EmployeeApiTest, never>
+        Osdk.Instance<EmployeeApiTest, never, PropertyKeys<EmployeeApiTest>>
+      >();
+
```

Comment:
Below I am testing if you had previously used explicit generics on `fetchPage` that it continues to work.